aries-core/src/js/components/helpers/Anchors/FooterLink.js

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef, useState } from 'react';
```

Comment:
This is maybe a bigger discussion for after the PR, but I think we need to assess all of the anchor treatments that currently exist in aries-core and make sure that we want the design system to support all of them and then potentially revisit earlier discussions about having them live under a single component called `Anchor` with prop `type`. Thoughts? @halocline

Reply:
@taysea Yes, I think the scope is larger than this PR. I added detail to this issue https://github.com/hpe-design/design-system/issues/308. Let's get on a call to talk through.